docs/root/operations/runtime.rst

```diff
@@ -6,3 +6,9 @@ Runtime
 :ref:`Runtime configuration <config_runtime>` can be used to modify various server settings
 without restarting Envoy. The runtime settings that are available depend on how the server is
 configured. They are documented in the relevant sections of the :ref:`configuration guide <config>`.
+
+Runtime flags are also used as a mechanism to disable new behavior or risky changes: such changes
```

Comment:
optional rephrase:
disable new behavior ->
to disable behavioral changes not otherwise guarded by configuration?